easybuild/easyconfigs/h/HEALPix/HEALPix-3.82-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'MakeCp'
```

Comment:
@jose-d Is there a particular reason you didn't use the existing custom easyblock for HEALPix, like the existing HEALPix easyconfigs do?

Reply:
@boegel yes, they changed/fixed their configure script.

in the past, the only way to install HEALPIX was usage of interactive  `configure`, having different interactive questions/answers, which were subjects of change every then and now.

I guess, handling the Q/A dialog was most likely the main reason to create custom easyblock.

With this release of Healpix it is finally possible to `configure` healpix with cli arguments only - so no interactive dialog with [Q/A](https://github.com/easybuilders/easybuild-easyblocks/blob/aa894d86baf653d9568c6d3841deb5b5747feca7/easybuild/easyblocks/h/healpix.py#L74C13-L74C13) needed anymore. 

So I think there is no reason to use the Q-A based easyblock anymore.